Sources/GRPC/AsyncAwaitSupport/GRPCAsyncRequestStreamWriter.swift

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 #if compiler(>=5.6)
+import DequeModule
```

Comment:
You are not using this here as far as I can see right?

Reply:
Indeed I'm not. I was at some point it seems but forgot to remove the import.
Is there a way for Xcode to let you know if you have unused imports?